database/database_connection.py

```diff
@@ -18,16 +20,17 @@ async def init_models(self):
             async with self.engine.begin() as conn:
                 await conn.run_sync(Base.metadata.create_all)
             self.engine.echo = True
+        except ConnectionRefusedError:
+            logging.fatal("Удаленный компьютер отклонил это сетевое подключение")
         except socket.gaierror:
-            print("Не удалось разрешить имя хоста в строке подключения к базе данных")
-            raise
+            logging.fatal("Не удалось разрешить имя хоста в строке подключения к базе данных")
+        except asyncpg.exceptions.ConnectionDoesNotExistError:
+            logging.fatal("Соединение было закрыто во время операции")
 
     async def async_init(self):
         await self.init_models()
 
 
-try:
-    db_manager = DatabaseManager(POSTGRES_CONNECTION)
-    asyncio.run(db_manager.init_models())
-except (OperationalError, socket.gaierror):
-    print("Не удалось инициализировать DatabaseManager")
+configure_logging()
+db_manager = DatabaseManager(POSTGRES_CONNECTION)
+asyncio.run(db_manager.init_models())
```

Comment:
What do you mean about this line and line (18) in main.py? How work python interpretator?